### ISSUE TITLE
[jshow] Slideshow images

### ISSUE THREAD

**Issue by miniwebs** [joomla/joomla-websites#137]
Alt tags not showing on slideshow images - recommend adding alt tags for accessibility. Perhaps showcase website title as Alt tag

https://joomla-showcase.octopoos-dev.com/browse-sites/www-sunliner-com-au.html#case

![showcase-image](https://cloud.githubusercontent.com/assets/827110/10409699/f2d475fe-6f6d-11e5-801c-2283cdfcf995.jpg)

**Comment by MAT978** [comment]
Hello,
The Joomla! Showcase Directory is still under development.
thanks

**Comment by pbwebdev** [comment]
This is from #joomladayau. Teaching group of people how to contribute to Joomla by reporting issues and test for accessibility.

Showcase team will pick them up from here and make the changes on the new dev site.

Thanks @MAT978

**Comment by mbabker** [comment]
Another item with no activity...

**Comment by oliviernolbert** [comment]
Website slideshow images have now a 'Alt' set with '#website_title# By #website_author#'.

Can be tested here: https://showcase.joomla.org/new/browse-sites/16359.html

**Comment by sebastienlapoux** [comment]
Thanks to check and to close this issue and/or to give your feedback.

**Comment by Sandra97** [comment]
I confirm the images of the homepage have now alt tags.
But other pages, Browse Sites for example, alt tags are there but containing currently only "####". I suppose you're currently working on? 
I noticed also in the detail view pages that the alt tags are currently missing (or not yet added).

**Comment by oliviernolbert** [comment]
Sandra,

Alt tags are now available in 'Browse Sites' page, sub-pages and detail view pages.

**Comment by designbengel** [comment]
But you could include the Site title fixed, no ? As @miniwebs said it´s for accessibility reasons important.

**Comment by sebastienlapoux** [comment]
Hi Elisa,
The alt is using Site Title + Site Creator. What're you speaking about please?

**Comment by designbengel** [comment]
Sorry, i misread it :-) I read "Alt tags are not available"

**Comment by sebastienlapoux** [comment]
:-)
@miniwebs thanks to check and close this issue if it's ok for you.

**Comment by Sandra97** [comment]
> Alt tags are now available in 'Browse Sites' page, sub-pages and detail view pages.

I confirm!
